components/Map/Map.jsx

```diff
@@ -177,6 +177,27 @@ class Map extends React.Component {
         map.once('idle', (e) => {
           this.setState({ mapReady: true });
         });
+
+				// Check to see if councilId is present and updates the state
+				const {
```

Comment:
This block doesn't seem to be used since it is only run on `map.on('load')`. No councils will ever be selected when the map loads (aka on page-load).

There is a possibility that we might want to include this when we implement query parameters that pre-configure the map (aka user pastes a url to the app that specifies `councild=1`, and map loads zoomed in on council 1).

Can we find a way to preserve this logic? Can we put it in a function, and then leave a comment here saying "run function X when query parameter logic is implemented". Let me know what you think.

Reply:
Agreed, I moved it over to its own method for future use.